### PR TITLE
Update cal-com-share-meeting-links extension

### PR DESCRIPTION
**Pull request by AlexIsMaking**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/cal-com-share-meeting-links/src/index.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default function Command() {
           key={item.id}
           title={item.title}
           accessories={getAccessories(item)}
-          subtitle={item.description}
+          keywords={item.length ? [item.length.toString()] : []}
```
Comment (pernielsentikaer) [inline extensions/cal-com-share-meeting-links/src/index.tsx:33]: I don't think you should remove `subtitle` completely and replace it with keywords.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/cal-com-share-meeting-links/src/index.tsx (hunk 1 above)
Comment (AlexIsMaking) [reply extensions/cal-com-share-meeting-links/src/index.tsx:34]: That's so strange, I didn't have that line in my copy of the extension 🤔 Anyway, that's fixed now.

Comment (AlexIsMaking) [reply extensions/cal-com-share-meeting-links/src/index.tsx:34]: Actually, I've just double checked and the version of the extension that's in the store now doesn't include the subtitle so maybe it's your version that's out of sync?

FWIW the subtitle looks a bit messy and isn't that useful, for me, as I already know what each event type is for, based its name.

With sub-titles (from my latest commit):

CleanShot 2023-12-05 at 09 35 28@2x

Without (from the store):

CleanShot 2023-12-05 at 09 36 13@2x

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**